### PR TITLE
feat: add example of title transfer

### PR DESCRIPTION
**Pull request by cavacado**
This pull request details a guided example of a title transfer.

dont merge until the cli commands have been merged.

feel free to comment and improve upon the document.

### CODE REVIEW

File: docs/overview/title-transfer-example.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+The token is held by `Bob`, token is owned by `Charlie`, no change in state from previous action.
+
+### Actions:
+1. Alice:
+    - no access
+2. Bob:
+    - change holder
+    - endorse transfer of ownership
+3. Charlie:
+    - nominate change of ownership (approve new owner)
```
Comment (simboonlong) [inline docs/overview/title-transfer-example.md:131]:
```suggestion
2. Bob:
    - change holder (approve new owner)
    - endorse transfer of ownership
3. Charlie:
    - nominate change of ownership
```

in a way, Bob decides whether to approve the new nominated owner by charlie. at this point charlie can still kinda suggest other ppl as new owner.

i thought the `(approve new owner)` text might misled, wdyt?

File: sidebars.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "docs": {
     "Introduction": ["introduction/what-is-tradetrust", "introduction/tradetrust-layer-boundaries"],
-    "Overview": ["overview/issuer-dns-txt-record", "overview/token-registry", "overview/title-transfer"],
+    "Overview": ["overview/issuer-dns-txt-record", "overview/token-registry", "overview/title-transfer", "overview/title-transfer-example"],
     "Technical": ["technical/open-attestation"],
```
Comment (simboonlong) [inline sidebars.json:5]: could nest this 
```suggestion
    "Overview": ["overview/issuer-dns-txt-record", "overview/token-registry", "overview/title-transfer", {
        "label": "Example",
        "type": "category",
        "items": [
          "overview/title-transfer/example",
        ]
      }"],
    "Technical": ["technical/open-attestation"],
```

i think

File: docs/overview/title-transfer-example.md (hunk 1 above)
Comment (cavacado) [reply docs/overview/title-transfer-example.md:131]: yup you are right, in this case maybe we should just get rid of the `(approve new owner)` text

File: sidebars.json (hunk 2 above)
Comment (cavacado) [reply sidebars.json:5]: k

Comment (simboonlong) [reply sidebars.json]: oops i meant sth like this: 

Screenshot 2021-09-02 at 4 11 00 PM

overview = the existing copy of title transfer background
title transfer example = the new docs in this pr

then can have subsequent `surrender example` topic later on

**Review by simboonlong — APPROVED** [review]
lgtm anyway

File: sidebars.json (hunk 2 above)
Comment (cavacado) [reply sidebars.json]: 👌

**Pull request merged**